.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 18.18.2
```

Comment:
Hi @jvallesm, we should remove this, it came while installing modules.

Reply:
ah sorry! it's a local file I added to quickly make it work, I didn't notice I included it 🙏